frontend/components/text-classifier/labeling-rules/RulesSummary.vue

```diff
@@ -0,0 +1,241 @@
+<template>
```

Comment:
Since we have no `summary` concept anymore. We can align naming and call the component as `RulesManagement` ?